maven.gradle

```diff
@@ -0,0 +1,35 @@
+rootProject.allprojects {
+    String rosMavenPath = System.getenv("ROS_MAVEN_PATH")
+    String rosMavenRepository = System.getenv("ROS_MAVEN_REPOSITORY")
+    repositories {
+        if (rosMavenPath != null) {
+            rosMavenPath.tokenize(":").each { path ->
+                //noinspection GroovyAssignabilityCheck
+                maven {
+                    // We can't use uri() here because we aren't running inside something
+                    // that implements the Script interface.
+                    url "file:${path}"
+                }
+            }
+        }
+        //noinspection GroovyAssignabilityCheck
+        maven {
+            url "http://repository.springsource.com/maven/bundles/release"
+        }
+        //noinspection GroovyAssignabilityCheck
+        maven {
+            url "http://repository.springsource.com/maven/bundles/external"
+        }
+        if (rosMavenRepository != null) {
+            //noinspection GroovyAssignabilityCheck
+            maven {
+                url rosMavenRepository
+            }
+        }
+        //noinspection GroovyAssignabilityCheck
+        maven {
+            url "https://github.com/rosjava/rosjava_mvn_repo/raw/master"
+        }
+        mavenCentral()
+    }
+}
```

Comment:
Is this just for convenience?

I'm reluctant to add this - means we have a maven list in two places (other being RosPlugin.groovy) that we have to maintain. And the plugin is better because it is much faster (doesn't need a fetch for every project configuration).